index.js

```diff
@@ -17,17 +17,23 @@ app.get('/auth/google', Auth.redirectToLogin)
 app.get('/auth/google/callback', Auth.handleResponse('/user/profile'))
 
 app.get('/user/profile', function (req, res) {
-  const {username, email} = req.user
-
-  res.send(`<h1>UserProfile</h1>
-<p>You are successful logged in as ${username}</p>
-<form>
-  <label for="email">Email:</label>
-  <input id="email" name="email" value="${email}"/>
-</form>
-${JSON.stringify(req.user)}`)
+  if (!req.user) {
+    res.redirect('/')
```

Comment:
probably want to return at this point, so we don't send content as well as a redirect.

eg. `return res.redirect('/')`